### DISCUSSION TITLE
Vernal equinox animation "day over year animation"

### DISCUSSION THREAD

**Discussion by ghost** [Stellarium/stellarium#661] [Q&A]
I would like a feature to create an animation with possibility to set "view direction, animation speed, time spann ***using annual date*** for example vernal equinox.

As i have understand it many ancient civilization did use EAST FACING and VERNAL EQUINOX to record events that took place. So the consteallation map was sort of a timekeeper.

Because if the ancient considered  "the age"? by facing east during vernal equinox and took  direction to some significant planet like neptune you can figure out the year pretty conclusively?

Could you not? I think Neptune some kind of time keeper relative the vernal equinox, is that right?

**Comment by alex-w** [comment]
I do not understand your problem. You may set view direction, animation speed and time span through scripting already. You may make visible the equinox points too.

**Comment by gzotti** [comment]
You can read time and/or date from the sky, yes. 
Neptune could not have been used by the Ancients. It is not visible to the unaided eye and was discovered in 1846. It passes the Vernal equinox every 164 years or so. The Sun is the timekeeper to count years with its run through the ecliptic.

**Comment by ghost** [comment]
Well i think it is pretty conclusively that the wandering stars, was depicted by gods riding chariots by both the ancient greeks and romans. That it was not visible to them by the naked eye does not mean it was not known to them, i think they even did know its precessional cycle of 164 years.

There is a heat cycle every 164 years, and it ain't farting driving it.
https://www.facebook.com/photo.php?fbid=2074633005939845&set=a.303634029706427&type=3&theater

**Comment by ghost** [comment]
So i can create an animation with just a single or a few frames per year? Set it at a specific date and have the animation laping over 164 years? You say script "can you set the parameters in program?".

I really would like an animation facing east at vernal equinox laping over the slowest moving planet Neptunus cycle. I am not good at astrology and want to know will Neptunus be roughly at same place relative the fixstars every 164<>166 year? 

Also the perhelion, yeah i know they say its orbit basicly round. If one would go with the heat trend every 164 year i would say its perhelion around 2025-2035
